cmd/releasego/testdata/publish-announcement/three-branches.golden.md

```diff
@@ -0,0 +1,17 @@
+---
+post_title: Go 1.23.1-1, 1.22.8-1 and 1.21.11-16 Microsoft builds now available.
+author1: 
+post_slug: go-1-23-1-1-1-22-8-1-and-1-21-11-16-microsoft-builds-now-available
```

Comment:
The slug is a bit confusing for 3 versions, ideally we need to add a separator to each version

Reply:
fixed!